contracts/axelar-gas-service/src/contract.rs

```diff
@@ -66,6 +66,30 @@ impl AxelarGasService {
         Ok(())
     }
 
+    /// Add additional gas payment after initiating a cross-chain call.
+    pub fn add_gas(
+        env: Env,
+        sender: Address,
+        message_id: String,
+        refund_address: Address,
+        token: Token,
```

Comment:
```suggestion
        token: Token,
        refund_address: Address,
```
match ordering from EVM gas service